appservice/src/deploy/runWithZipStream.ts

```diff
@@ -71,15 +71,22 @@ const commonGlobSettings: {} = {
 /**
  * Adds files using glob filtering
  */
-async function getFilesFromGlob(folderPath: string): Promise<string[]> {
+async function getFilesFromGlob(folderPath: string, client: SiteClient): Promise<string[]> {
     const zipDeployConfig: vscode.WorkspaceConfiguration = vscode.workspace.getConfiguration(ext.prefix, vscode.Uri.file(folderPath));
     const globOptions = { cwd: folderPath, followSymbolicLinks: true, dot: true };
     const globPattern: string = zipDeployConfig.get<string>('zipGlobPattern') || '**/*';
     const filesToInclude: string[] = await globby(globPattern, globOptions);
+    const zipIgnorePatternStr = 'zipIgnorePattern';
 
-    const ignorePattern: string | string[] = zipDeployConfig.get<string | string[]>('zipIgnorePattern') || '';
+    const ignorePattern: string | string[] = zipDeployConfig.get<string | string[]>(zipIgnorePatternStr) || '';
     const filesToIgnore: string[] = await globby(ignorePattern, globOptions);
 
+    ext.outputChannel.appendLog(`Ignoring files from \"${ext.prefix}.${zipIgnorePatternStr}\"`, { resourceName: client.fullName });
+
+    for (const file of filesToIgnore) {
+        ext.outputChannel.appendLine(`\"${file}\"`);
```

Comment:
Can we get an example of what this looks like? If I ignore the "node_modules" folder - it's not gonna print out every file in "node_modules", is it?

Reply:
Yes, with the last commit that's what was happening, but I fixed it with the latest one. 

With the recent commit this is what it looks like...
![image](https://user-images.githubusercontent.com/17537419/116333058-b2f8b780-a787-11eb-89e9-3fc4deac51fb.png)